source/translators/quickstarts/start-translating-openedx.rst

```diff
@@ -101,3 +101,10 @@ Next Steps
   `docs.opened.org Github repository`_
 
 .. _docs.opened.org Github repository: https://github.com/openedx/edx-documentation
+
+Useful links
+**********
+
+	* Confluence site for Translation Working Group: https://openedx.atlassian.net/wiki/spaces/COMM/pages/3157524644/Translation+Working+Group
+	* Open edX WG website link: https://openedx.org/open-edx-community-working-groups/
```

Comment:
```suggestion
* `Confluence site for Translation Working Group <https://openedx.atlassian.net/wiki/spaces/COMM/pages/3157524644/Translation+Working+Group>`_

* `Open edX WG website link <https://openedx.org/open-edx-community-working-groups/>`_
```

The bullet points shouldn't be tabbed in and we can make the links more standard RST links for cleanliness.